### PR TITLE
Feature/websocket

### PR DESCRIPTION
**Pull request by ohler55**
Adds a WebSocket example and fixes an unsubscribe bug.

### CODE REVIEW

**Comment by jduhamel** [comment]
missing price.graphql for the exmaple to build.

**Comment by jduhamel** [comment]
LGTM +1

File: examples/interface/go.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module song
 
 go 1.15
 
-require github.com/uhn/ggql v1.1.0
+require github.com/uhn/ggql v1.2.4
```
Comment (jduhamel) [inline examples/interface/go.mod:5]: Shouldn't this be 1.2.5?

File: examples/reflection/go.mod [hunk 2]
```diff
@@ -2,4 +2,4 @@ module song
 
 go 1.15
 
-require github.com/uhn/ggql v1.1.0
+require github.com/uhn/ggql v1.2.4
```
Comment (jduhamel) [inline examples/reflection/go.mod:5]: Shouldn't this be 1.2.5?

File: examples/root/go.mod [hunk 3]
```diff
@@ -2,4 +2,4 @@ module song
 
 go 1.15
 
-require github.com/uhn/ggql v1.1.0
+require github.com/uhn/ggql v1.2.4
```
Comment (jduhamel) [inline examples/root/go.mod]: Shouldn't this be 1.2.5?

**Review by jduhamel — CHANGES_REQUESTED** [review]

File: examples/reflection/go.mod (hunk 2 above)
Comment (ohler55) [reply examples/reflection/go.mod]: done

File: examples/root/go.mod (hunk 3 above)
Comment (ohler55) [reply examples/root/go.mod]: done

**Review by jduhamel — APPROVED** [review]

File: examples/interface/go.mod (hunk 1 above)
Comment (ohler55) [reply examples/interface/go.mod]: done

**Pull request merged**